featurebyte/routes/batch_feature_table/api.py

```diff
@@ -146,3 +147,20 @@ async def download_table_as_pyarrow_table(
         get_credential=request.state.get_credential,
     )
     return result
+
+
+@router.patch("/{batch_feature_table_id}/description", response_model=BatchFeatureTableModel)
+async def update_batch_feature_table_description(
```

Comment:
seems like we mostly duplicate this logic across all the routers - i wonder if it's possible to reduce boilerplate by putting router paths into a base class, and having child routers inherit from that.

maybe sth like https://stackoverflow.com/questions/63853813/how-to-create-routes-with-fastapi-within-a-class?

Reply:
Great idea.  Will look into this in a separate PR since this one is rather large.